### PR TITLE
Raised to play 2.5.3

### PR DESCRIPTION
**Pull request by breadfan**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,7 +2,7 @@ organization:= "com.kenshoo"
 
 name := "metrics-play"
 
-version := "2.4.0_0.4.1"
+version := "2.5.0_0.5.0-play-2.5-fix"
 
 scalaVersion := "2.11.6"
```
Comment (halfninja) [inline build.sbt]: Play 2.5 drops Scala 2.10 so this could be removed from crossScalaVersions.

**Comment by aakavalevich** [comment]
Who can merge the PR? I would be really nice to have Play 2.5 support.

**Comment by breadfan** [comment]
Since there seems to be no response from the originator I've created a fork and I'm just about setting up a travis build and deployment.

**Comment by breadfan** [comment]
Here it is: https://github.com/breadfan/metrics-play

**Comment by aakavalevich** [comment]
has it been already released?

**Comment by breadfan** [comment]
Yes...add my bintray repo ...i think jcenter takes some time to replicate
Am 12.07.2016 15:41 schrieb "Aliaksandr Kavalevich" <
notifications@github.com>:

**Comment by breadfan** [comment]
It's now available via jcenter: http://jcenter.bintray.com/de/threedimensions/metrics-play_2.11/2.5.13/

**Comment by mosheeshel** [comment]
Stale PR, closing
@rannissim FYI

**Pull request closed without merging**